analysis/selectDiphotonEvents.cpp

```diff
@@ -376,19 +419,31 @@ int main(int argc, char* argv[])
     auto photon1   = event->GetPhysObjects(EPhysObjType::kGoodPhoton)[0];
     auto photon2   = event->GetPhysObjects(EPhysObjType::kGoodPhoton)[1];
     auto caloTower = event->GetPhysObjects(EPhysObjType::kCaloTower);
-
+    //Vertex info//Date:09/02/2023
+    auto vertexs = event->GetPhysObjects(EPhysObjType::kVertex);
```

Comment:
The plural of "vertex" is "vertices" or (much, much less popular) "vertexes" ;)

Reply:
Thanks @jniedzie, I know it. I just randomly named it. I have changed it now.